.github/workflows/build.yml

```diff
@@ -17,6 +17,14 @@ jobs:
       - name: Install packages
         run: pip install -r requirements_dev.txt
 
+      - name: Lint
+        run: |
+          # Imports order
+          isort --check-only --diff .
+
+          # Code format
+          black --check .
```

Comment:
@dan8f these two will conflict each other if you don't tell isort to use black config
https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

Reply:
`isort --profile black` when you apply it and when you check too

Reply:
It was added in the `pyproject.toml` file

`tool.isort]
profile = "black"`

Reply:
ah, that should be fine, I missed it